lib/ansible/modules/packaging/os/yum.py

```diff
@@ -347,7 +348,7 @@ def is_group_env_installed(name, conf_file, installroot='/'):
     name_lower = name.lower()
 
     my = yum_base(conf_file, installroot)
-    if yum.__version__ >= '3.4':
+    if LooseVersion(yum.__version__) >= LooseVersion('3.4'):
```

Comment:
LooseVersion is broken in Python 3

Reply:
@abadger any hints on what is proper way of doing this?

Reply:
For yum, use ```yum.__version_info__``` to compare against.